query.go

```diff
@@ -59,7 +59,9 @@ type Query interface {
 	// SetConsistency sets the consistency level for this query.
 	SetConsistency(c gocql.Consistency)
 
-	// SetConsistency sets the consistency level for this query.
+	// Consistency sets the consistency level for this query. If no consistency
+	// level have been set, the default consistency level of the cluster
```

Comment:
have -> has

Reply:
I've copied it from gcql. anyway. updated